core/src/providers/anthropic.rs

```diff
@@ -79,7 +81,119 @@ impl AnthropicLLM {
             .parse::<Uri>()?)
     }
 
-    async fn streamed_completion(
+    fn chat_prompt(&self, messages: &Vec<ChatMessage>) -> String {
+        let mut prompt = messages
+            .iter()
+            .map(|cm| -> String {
+                format!(
+                    "\n\n{}: {}",
+                    match cm.role {
+                        ChatMessageRole::System => "Assistant",
```

Comment:
There's a bit question on what to do here with System message. It's not really the assistant talking, it's not really the human talking. If we had to choose I would probably use "Human" as the initial chat block instructions prompt will be mapped to a "System" message and since these are instructions they are better mapped to the Human than the Assistant.

Proposal:
- First step, use Human for System and eventually run some test to convince ourselves that it's OK (we'll see it quickly by comparing Chat response between ChatGPT 3.5 and Claude)